File: Day 29/JatinSaini664 (3).java
import java.util.*;

public class JatinSaini664{
    public static int countWords(String s){
        int words=0, i=0;
        while(i<s.length()){
            if(s.charAt(i)==' '){
                words++;
            }
            i++;
        }
        if(s.charAt(i-1)!=' ')
            words++;
        return words;
    }
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        String s = scn.nextLine();
        System.out.println(countWords(s));
    }
}

File: Day 29/Jatin-Saini664 (3).cpp
#include<bits/stdc++.h>

using namespace std;

int countWords(string s){
    int words=0, i=0;
    while(i<s.size()){
        if(s[i]==' '){
            words++;
        }
        i++;
    }
    if(s[i-1]!=' ')
        words++;
    return words;
}

int main(){
    string s;
    getline(cin, s);
    cout<<countWords(s)<<" ";
}